make stan/math/prim/scal/meta/container_view.hpp-test fails

## Bug Report

__Version__
develop

__Description__
test failure

__Steps__
`make stan/math/prim/scal/meta/container_view.hpp-test`

__Output__
```
In file included from <command-line>:0:0:
./stan/math/prim/scal/meta/container_view.hpp: In member function ‘stan::math::container_view<stan::math::dummy, T2>::scalar_t stan::math::container_view<stan::math::dummy, T2>::operator[](int) const’:
./stan/math/prim/scal/meta/container_view.hpp:82:17: error: ‘out_of_range’ is not a member of ‘std’
           throw std::out_of_range("can't access dummy elements.");
                 ^
make/tests:64: recipe for target 'stan/math/prim/scal/meta/container_view.hpp-test' failed
make: *** [stan/math/prim/scal/meta/container_view.hpp-test] Error 1
```

__Expected Output__
Nothing

__Additional Information__
Works with clang++ and g++-5 but not g++-4.9

--- a/stan/math/prim/scal/meta/container_view.hpp
+++ b/stan/math/prim/scal/meta/container_view.hpp
@@ -1,8 +1,8 @@
 #ifndef STAN_MATH_PRIM_SCAL_META_CONTAINER_VIEW_HPP
 #define STAN_MATH_PRIM_SCAL_META_CONTAINER_VIEW_HPP
 
-#include <stdexcept>
 #include <stan/math/prim/scal/meta/scalar_type.hpp>
+#include <stdexcept>
 
 namespace stan {
 

--- a/stan/math/prim/scal/meta/container_view.hpp
+++ b/stan/math/prim/scal/meta/container_view.hpp
@@ -1,6 +1,7 @@
 #ifndef STAN_MATH_PRIM_SCAL_META_CONTAINER_VIEW_HPP
 #define STAN_MATH_PRIM_SCAL_META_CONTAINER_VIEW_HPP
 
+#include <stdexcept>
 #include <stan/math/prim/scal/meta/scalar_type.hpp>
 
 namespace stan {

